content/study/posts/congo-mb.json

```diff
@@ -1,5 +1,12 @@
 {
   "version": 8,
+  "sources": {
+    "35kv": {
+      "tiles": [
+        "https://aep-tiles.staging.derilinx.com/tiles/ke/35kv/{z}/{x}/{y}.pbf"
```

Comment:
These are tiles for Kenya. When the user enables the layer, nothing will show over Congo.

Reply:
Oops. I updated it with Congo dataset.